File: src/main/java/no/nav/tjenestepensjon/simulering/service/SimulerPensjonServiceImpl.java
package no.nav.tjenestepensjon.simulering.service;

import no.nav.tjenestepensjon.simulering.domain.TPOrdning;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.stereotype.Component;

import java.util.List;

@Component
public class SimulerPensjonServiceImpl implements SimulerPensjonService {

    private static final Logger LOG = LoggerFactory.getLogger(SimulerPensjonServiceImpl.class);

    @Override
    public SimulerPensjonResponse simulerPensjon(List<TPOrdning> tpOrdningList, TPOrdning tpLatest) {

        return new SimulerPensjonResponse(null, null);
    }
}

File: src/main/kotlin/no/nav/tjenestepensjon/simulering/service/SimulerPensjonServiceImpl.kt
package no.nav.tjenestepensjon.simulering.service

import no.nav.tjenestepensjon.simulering.model.v1.domain.TPOrdning
import org.slf4j.LoggerFactory
import org.springframework.stereotype.Component

@Component
class SimulerPensjonServiceImpl : SimulerPensjonService {
    override fun simulerPensjon(tpOrdningList: List<TPOrdning>, tpLatest: TPOrdning): SimulerPensjonResponse {
        return SimulerPensjonResponse(null, null)
    }

    companion object {
        @JvmStatic
        @Suppress("JAVA_CLASS_ON_COMPANION")
        private val LOG = LoggerFactory.getLogger(javaClass.enclosingClass)
    }
}